src/pages/Dashboard/SubscribedServices.tsx

```diff
@@ -92,6 +92,58 @@ export const SubscribedServices = () => {
                     </Column>
                   </Card>
                 ))}
+                <Card
+                  key={99388474}
+                  style={{
+                    background: '#EDF6F8',
+                    minWidth: '92px',
+                    marginRight: '2.5%',
+                    height: '100%',
+                  }}
+                >
+                  <Column
+                    justifyContent="center"
+                    alignItems="center"
+                    fullHeight
+                  >
+                    <Text
+                      casing="titleCase"
+                      size={12}
+                      color={Colors.darkGreen}
+                      weight={700}
+                      alignment="center"
+                    >
+                      Test 1
```

Comment:
This looks like dummy content. Can I merge like this?

Reply:
The dummy content has been removed.